### DISCUSSION TITLE
glb model texture uv-mapping issue

### DISCUSSION THREAD

**Discussion by Nwjwifh** [google/model-viewer#4176] [Q&A]
Hello, I tried to uv-map png image on several glb models and change texture, but it doesn't work(picture 1). 
Actually, I think color of image is applied, but I want complete image file wrapped on model(like picture2). Please help me
<picture1>
<img width="900" alt="prob1" src="https://user-images.githubusercontent.com/111439484/226814588-633309a8-aec5-4cf1-a5dd-f800f4848e04.PNG">

<picture2>
<img width="841" alt="prob2" src="https://user-images.githubusercontent.com/111439484/226814605-887e1d73-3281-4e41-87dc-620aa153c2ce.PNG">

**Comment by elalish** [comment]
It looks like the UV coordinates (or texture transform) of the model is set to only use a small corner of the texture. You'll need to check in proper 3D authoring software like Blender.

**Reply by Nwjwifh** [reply]
Thanks for the answer. But is there any way of fixing this problem by editing gltf's json code?(without using software program)

**Reply by elalish** [reply]
You can probably fix it with a texture transform.
